src/otx/cli/cli.py

```diff
@@ -117,6 +118,11 @@ def engine_subcommand_parser(**kwargs) -> ArgumentParser:
             type=str,
             help="The metric to monitor the model performance during training callbacks.",
         )
+        parser.add_argument(
+            "--no-update-num-classes",
+            help="This disables the ability to update the model with the num_classes in the dataset.",
+            action="store_true",
```

Comment:
Can we enhance this for the better clarification?
```suggestion
            "--disable-infer-num-classes",
            help="OTX automatically infers num_classes from the given dataset and applies it to the model initialization. Consequently, there might be a mismatch with the provided model configuration during runtime. Setting this option to true will disable this behavior.",
            action="store_true",
```

Reply:
Thanks for the great suggestion. - https://github.com/openvinotoolkit/training_extensions/pull/2861/commits/6125aa69007b96e46b89227d882ae18f4ffe2ad7